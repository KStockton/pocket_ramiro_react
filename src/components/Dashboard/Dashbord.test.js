import React from 'react';
import { shallow } from 'enzyme'
import { Dashboard, mapDispatchToProps } from './';

describe('TicketDashboard', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Dashboard getActiveTickets={jest.fn()}/>);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

describe('MapDispatchToProps', () => {
  let mockDispatch, mappedProps;

  beforeEach(() => {
    mockDispatch = jest.fn();
    mappedProps = mapDispatchToProps(mockDispatch);
  });
});