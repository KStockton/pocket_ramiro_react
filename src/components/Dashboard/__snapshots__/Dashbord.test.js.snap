// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should match the snapshot 1`] = `
<main>
  <withRouter(Connect(CardContainer)) />
  }
</main>
`;

exports[`Dashboard should match the snapshot if location is resources 1`] = `
<main>
  <withRouter(Connect(CardContainer)) />
  }
</main>
`;
